rdkit/Dbase/DbConnection.py

```diff
@@ -337,12 +322,11 @@ def AddColumn(self, tableName, colName, colType):
     c = self.GetCursor()
     try:
       c.execute("alter table %s add %s %s" % (tableName, colName, colType))
-    except Exception:
+    except Exception:  # pragma: nocover
```

Comment:
just to get it off my chest (which I am sure to do again at some point): this type of `pragma` just pollutes the code without adding any actual value. I would prefer that they not be there. But I will live with them if you think the ugliness in the code is worth whatever value you get out of them